cmd/agent/common/helpers_nix.go

```diff
@@ -0,0 +1,30 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build freebsd netbsd openbsd solaris dragonfly linux darwin
+
+package common
+
+import (
+	"os"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+)
+
+const (
+	procfsPathEnv = "HOST_PROC"
```

Comment:
why pull the value from `HOST_PROC`? Is it a standard? Unless it is (and unless there are other reasons I'm missing), I think we should keep using the standard `DD_PROCFS_PATH` env var.

Reply:
it's not standard it's what they've implemented in `gopsutil` and the only env var that is honored.

Reply:
ok, thanks for the explanation. I'm not sure we actually want to override the "proc path" of gopsutil (=> all core system checks) in containerized environments through the same configuration option.

As a first approach, we could stick to only setting the value of the agent-configured `procfs_path` on python's psutil to solve this issue: https://github.com/DataDog/datadog-agent/issues/1511

Let's bring Kenafeh in the loop to discuss what the behavior of the core system checks should be

Reply:
(fwiw, Agent 5 uses the configured value of `procfs_path` for 3 core system checks: `file_handles`, `load` and `memory` ; but `cpu` for instance always uses the default proc path)

In general, it'd be good to try and keep this config option configurable at the integration-level as much as possible, unless there are library constraints (ex.: checks sharing usage of `psutil` or `gopsutil` which only accepts a library-wide option) that force us to use agent-wide options